e2e/playwright/search/src/tests/search-filters-date.e2e.ts

```diff
@@ -0,0 +1,120 @@
+/*!
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test, NodesApi, TrashcanApi, FileActionsApi, TEST_FILES } from '@alfresco/playwright-shared';
+
+test.describe('Search - Filters - Date', () => {
+  let nodesApi: NodesApi;
+  let trashcanApi: TrashcanApi;
+
+  const randomId = Utils.random();
+  const username = `user-${randomId}`;
+  const fileNamePdfKb = `${randomId}-fileNamePdf.pdf`;
+  const fileNameJpgMb = `${randomId}-fileNameJpg.jpg`;
+  const getCurrentAndPreviousDay = Utils.getCurrentAndPreviousDay();
```

Comment:
could you please rename that const to noun? It starts from 'get' word what suggest this const is a function but it isn't function because it is object returned by "getCurrentAndPreviousDay" which is real function. 

You can name it for example "currentAndPreviousDay"